ansible/inventory.yml

```diff
@@ -1,13 +1,17 @@
 all:
-  hosts:
+  children:
     dev:
+      hosts:
+        dev-1
     stage:
+      hosts:
+        stage-1
     prod:
+      hosts:
+        prod-1
     registry:
+      hosts:
+        registry-1
```

Comment:
Just wondering... why the addition of `-1` to the hosts here? (and related changes)

Is it about potentially handling multiple dev/stg/prod clusters? I fail to see the use case here though...

Reply:
@codificat I moved the host_vars to a group_vars.  I thought it would be confusing if the group and hosts have the same name...I don't think you can do it either.  I also wanted the ability to use `ansible-vault` on the `openshift_username` and `openshift_password`.

Example: https://github.com/jcpowermac/jenkins-on-openshift/tree/ansible_updates_testing/ansible/host_vars